R/select_rules.R

```diff
@@ -7,46 +7,63 @@
 #' Stability Selection).
 #'
 #' @param rules_matrix The rules matrix.
-#' @param rules_list A list of rules.
+#' @param rules A list of rules.
```

Comment:
For consistency in documentation, always choose either `A` or `The`. My recommendation is to use `A` throughout.